2996_reflection/reflection.md

```diff
@@ -3021,6 +3021,28 @@ Change the grammar for `$operator-or-punctuator$` in paragraph 1 of [lex.operato
 ```
 :::
 
+### [basic.def.odr]{.sref} One-definition rule {-}
+
+Preprend before paragraph 14 of [basic.def.odr]{.sref}:
```

Comment:
15 now?

Reply:
I thought CWG doesn't want us to renumber paragraphs?  They do that at editing time?

Reply:
I mean the paragraph that you're editing is paragraph 15 right now: http://eel.is/c++draft/basic.def.odr#15